bigtop-deploy/puppet/hieradata/bigtop/cluster.yaml

```diff
@@ -191,6 +191,7 @@ hadoop::common::tez_jars: "/usr/lib/tez"
 #kafka
 kafka::server::port: "9092"
 kafka::server::zookeeper_connection_string: "%{hiera('bigtop::hadoop_head_node')}:2181"
+kafka::server::bind_addr: ""
```

Comment:
We were [instructed](https://issues.apache.org/jira/browse/BIGTOP-2458?focusedCommentId=15334314&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15334314) not to put default values in `cluster.yaml`.